enterprise/reporting/error_extractor.go

```diff
@@ -109,48 +109,77 @@ func (ext *ExtractorHandle) getSimpleMessage(jsonStr string) string {
 	}
 
 	for key, erRes := range jsonMap {
-		erResStr, isString := erRes.(string)
+		if result := ext.handleKey(key, erRes); result != nil {
+			return *result
+		}
+	}
+	return ""
+}
+
+func (ext *ExtractorHandle) handleKey(key string, value interface{}) *string {
+	valueStr, isString := value.(string)
```

Comment:
GO's ok idiom. https://www.devzery.com/post/mastering-ok-in-golang-your-ultimate-guide
```suggestion
	valueStr, ok := value.(string)
```

Reply:
addressed